ozaria/site/components/interactive/PageInteractive/draggableOrdering/index.vue

```diff
@@ -34,12 +39,10 @@
       return {
         draggableGroup: Math.random().toString(),
 
-        promptSlots: [
-          { id: 1, text: 'one option' },
-          { id: 2, text: 'two option' },
-          { id: 3, text: 'three option' },
-          { id: 4, text: 'four option' }
-        ],
+        promptSlots: this.localizedInteractiveConfig.elements.map(option => ({
```

Comment:
Nitpick: `(this.localizedInteractiveConfig || {}).elements.map`

Reply:
As is it required this check shouldn't  be required.